### PR TITLE
added tutorial 06 and modified reductionProblem templates added unste…

### PR DESCRIPTION
**Pull request by sokratiag**
…adyNST and reducedUnsteadyNST

### CODE REVIEW

File: src/problems/reductionProblem/reductionProblemTemplates.C [hunk 1]
```diff
@@ -81,4 +80,39 @@ void reductionProblem::computeLift(T& Lfield, T& liftfield, T& omfield)
 	}
 }
 
+template<typename T>
+void reductionProblem::computeLiftT(T& Lfield, T& liftfield, T& omfield)
+{
+	scalar t_bc;
+	scalar area;
+
```
Comment (giovastabile) [inline src/problems/reductionProblem/reductionProblemTemplates.C]: remove unecessary returns

File: src/problems/unsteadyNST/NLsolve.H [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+#include "CourantNo.H"
+
+ runTime++;
+
+  Info << "Time = " << runTime.timeName() << nl << endl;
+
+  // --- Pressure-velocity PIMPLE corrector loop
+		// --- Pressure-velocity PISO corrector loop
+		//while (runtime.loop())
+		//{
```
Comment (giovastabile) [inline src/problems/unsteadyNST/NLsolve.H]: check carefully the indentation

File: src/problems/unsteadyNST/unsteadyNST.H [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    scalar finalTime;
+
+    /// Time step of the simulation
+    scalar timeStep;
+
+    /// Time step of the writing procedure
+    scalar writeEvery = timeStep;
+
+    /// Auxiliary variable to store the next writing instant
+
```
Comment (giovastabile) [inline src/problems/unsteadyNST/unsteadyNST.H]: remove the return otherwise doxygen will not work

File: src/problems/unsteadyNST/unsteadyNST.H [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    /// List of pointers used to form the temperature snapshots matrix
+    PtrList<volScalarField> Tfield;
+    /// Temperature field
+    autoPtr<volScalarField> _T;
+    /// List of pointers used to form the temperature modes
+    
+    PtrList<volScalarField> LTmodes;
+
+    /// List of pointers used to form the temperature modes
+    // PtrList<volScalarField> Tmodes;
```
Comment (giovastabile) [inline src/problems/unsteadyNST/unsteadyNST.H]: remove the comment

File: src/problems/unsteadyNST/unsteadyNST.H [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+    ///
+    /// @param[in]  folder     The folder used to save the reduced matrices.
+    /// @param[in]  NUmodes    The number of velocity modes.
+    /// @param[in]  NPmodes    The number of pressure modes.
+    /// @param[in]  NPmodes    The number of pressure modes.
+    /// @param[in]  NSUPmodes  The number of supremizer modes, 0 by default and it must remain always zero.
+    ///
+    void projectSUP(fileName folder, label NUmodes, label NPmodes, label NTmodes, label NSUPmodes);
+
+    List< Eigen::MatrixXd > convective_term_temperature(label NUmodes, label NTmodes, label NSUPmodes);
```
Comment (giovastabile) [inline src/problems/unsteadyNST/unsteadyNST.H]: add documentation comments

File: src/reducedProblems/reducedUnsteadyNST/reducedUnsteadyNST.H [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+
+    /// Method to reconstruct a solution from an online solve with a supremizer stabilisation technique.
+    /// stabilisation method
+    ///
+    /// @param      problem     a steadyNS full order problem.
+    /// @param[in]  folder      The folder where you want to store the results (default is "./ITHACAOutput/online_rec").
+    /// @param[in]  printevery  Variable to recover only every printevery online solutions default is 1.
+    ///
+    void reconstruct_sup(unsteadyNST& problem, fileName folder = "./online_rec", int printevery = 1);
+
```
Comment (giovastabile) [inline src/reducedProblems/reducedUnsteadyNST/reducedUnsteadyNST.H]: add documentation

File: tutorials/06NonIsothermalMixingElbow/06NonIsothermalMixingElbow.C [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+
+    // Set the online velocity
+    Eigen::MatrixXd vel_now;
+    vel_now.resize(2,1);
+    vel_now << 0.6, 1.2;
+    // Set the online temperature and the value of the internal field 
+    Eigen::MatrixXd temp_now;
+    temp_now.resize(3,1);
+    temp_now << 60, 70, 60;
+
```
Comment (giovastabile) [inline tutorials/06NonIsothermalMixingElbow/06NonIsothermalMixingElbow.C]: change to english according to tutorial4

File: src/reducedProblems/reducedUnsteadyNST/reducedUnsteadyNST.C [hunk 8]
```diff
@@ -0,0 +1,450 @@
...
+
+// * * * * * * * * * * * * * * * Constructors * * * * * * * * * * * * * * * * //
+
+// Constructor
+reducedUnsteadyNST::reducedUnsteadyNST()
+{
+
+}
+
+reducedUnsteadyNST::reducedUnsteadyNST(unsteadyNST& problem, word tipo)
```
Comment (giovastabile) [inline src/reducedProblems/reducedUnsteadyNST/reducedUnsteadyNST.C]: change to english, my fault

File: src/reducedProblems/reducedUnsteadyNST/reducedUnsteadyNST.C [hunk 9]
```diff
@@ -0,0 +1,450 @@
...
+{
+    B_matrix  = problem.B_matrix;
```
Comment (giovastabile) [inline src/reducedProblems/reducedUnsteadyNST/reducedUnsteadyNST.C]: check how it is implemented now for reducedSteadyns and do the same

**Comment by mtezzele** [comment]
I would suggest to close this and open a new PR with a proper history. You should not merge the master into your branch, instead you should fetch the master and then rebase.

**Pull request closed without merging**